docs/serverless-v2/08-04-trigger-lambda-by-event.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
You rather trigger sth with the else so I would change the title into -> `08-04-trigger-lambda-with-event.md`